Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -135,6 +135,40 @@ public final class PropertyVisitor: SyntaxVisitor {
     return typeSyntax.typeDescription
   }
 
+  private func findParamadigm(from node: VariableDeclSyntax) -> PropertyInfo.Paradigm {
+    switch findPropertyType(from: node) {
+    case .constant:
+      if let initializerDescription = findInitializerDescription(from: node.bindings) {
+        return .definedConstant(initializerDescription)
+      }
+      else {
+        return .undefinedConstant
+      }
+    case .variable:
+      if let initializerDescription = findInitializerDescription(from: node.bindings) {
+        return .definedVariable(initializerDescription)
+      }
+      else if let codeBlockDescription = findCodeBlockDescription(from: node.bindings) {
+        return .computedVariable(codeBlockDescription)
+      }
+      else {
+        return .undefinedVariable
+      }
+    }
+  }
+
+  private func findInitializerDescription(from node: PatternBindingListSyntax) -> String? {
+    let initializerClauseSyntaxes = node.compactMap { $0.initializer }
+    assert(initializerClauseSyntaxes.count <= 1, "A property should have at most one initializer.")
```

Comment:
not sure if we care, but this will likely assert if you write code that won't compile. Example:
```
let foo = 5 = 3
```

Reply:
I will add a test and we can check.

Reply:
Bizarre, but no assertion. https://github.com/fdiaz/SwiftInspector/pull/110/commits/1fbea486dc08f259ea7dcc8dbf1f72e1f446775d

Reply:
Interesting 🤔